om4labs/diags/moc/moc.py

```diff
@@ -157,17 +185,36 @@ def arguments(cliargs=None):
         "-t", "--topog", type=str, default=None, help="Path to topog file"
     )
 
-    return parser.parse_args(cliargs)
+    if template is True:
```

Comment:
I don't understand the template purpose

Reply:
The template functionality returns a generic dict() of options.  Dora can modify that dict() and use it to call run() directly.